tests/utils.go

```diff
@@ -3797,7 +3801,7 @@ func newDeleteHostDisksJob(diskPath string) *k8sv1.Pod {
 }
 
 func RenderHostPathJob(jobName string, dir string, hostPathType k8sv1.HostPathType, mountPropagation k8sv1.MountPropagationMode, cmd []string, args []string) *k8sv1.Pod {
-	job := RenderJob(jobName, cmd, args)
+	job := RenderPod(jobName, cmd, args)
```

Comment:
I am confused now. The function name has `Job` but it used `pod` rendering and the variable is `job`.

Reply:
I see. I will also change the names of the functions so it don't lie.